spec/index.md

```diff
@@ -5102,6 +5234,7 @@ RM.origin = FromCanister {
   }
 S.call_contexts[Ctxt_id].deleted
 S.call_contexts[Ctxt_id].canister ∈ dom(S.balances)
+RM.deadline ≠ Expired
```

Comment:
I guess we need an analogous transition for expired messages that just drops them without any refund.